### PR TITLE
Log the sentinels in the child process rather than the proxy process

### PR DESCRIPTION
**Pull request by markusthoemmes**
To reflect the changes from nimbella/openwhisk-runtime-go#4.

### CODE REVIEW

File: core/php8.0Action/runner.php [hunk 1]
```diff
@@ -85,8 +86,8 @@
     }
 
     // ensure that the sentinels will be on their own lines
-    file_put_contents('php://stderr', "\n");
-    file_put_contents('php://stdout', "\n");
+    file_put_contents('php://stderr', "\n" . $sentinel);
+    file_put_contents('php://stdout', "\n" . $sentinel);
```
Comment (markusthoemmes) [inline core/php8.0Action/runner.php:90]: Note: I believe this newline is causing us to write "unnecessary" logs, but I wanted to retain the existing behavior in this PR. If we find that we want to change the behavior, we should do so separately.

Comment (rabbah) [reply core/php8.0Action/runner.php:90]: it covers a corner case? if function logs without a new line, the sentinel gets mangled as part of that log line (this is likely the case everywhere else).

**Review by rabbah — COMMENTED** [review]
lgtm modulo the buffer flush.

**Comment by markusthoemmes** [comment]
I'm fairly sure that flushing isn't necessary (or even possible) because `file_put_contents` opens the file, writes to it and then closes it. I'd expect the close operation to ensure all data is flushed.

**Review by rabbah — APPROVED** [review]

**Pull request merged**